fitbenchmarking/fitbenchmarking
# Fixing status flag for scipy

#### Description of Work

Fixes #627 

Fixes incorrect flag

#### Testing Instructions

1. check that the status reported here https://github.com/scipy/scipy/blob/01d8bfb6f239df4ce70c799b9b485b53733c9911/scipy/optimize/optimize.py#L762-L773 is corresponds to our flag

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
